testing/e2e/skr-tester/pkg/command/assert.go

```diff
@@ -135,6 +146,110 @@ func (cmd *AssertCommand) Run() error {
 			}
 		}
 		fmt.Println("All specified admins are found in cluster role bindings")
+	} else if cmd.btpManagerSecretExists { //
+		kubeconfig, err := kcpClient.GetKubeconfig(cmd.instanceID)
+		if err != nil {
+			return fmt.Errorf("failed to get kubeconfig: %v", err)
+		}
+		cmd.checkBTPManagerSecret(kubeconfig)
+	} else if cmd.deleteBtpManagerSecret {
+		kubeconfig, err := kcpClient.GetKubeconfig(cmd.instanceID)
+		if err != nil {
+			return fmt.Errorf("failed to get kubeconfig: %v", err)
+		}
+		restCfg, err := clientcmd.RESTConfigFromKubeConfig(kubeconfig)
+		if err != nil {
+			return fmt.Errorf("while creating rest config from kubeconfig: %w", err)
+		}
+		k8sCli, err := client.New(restCfg, client.Options{
+			Scheme: scheme.Scheme,
+		})
+		if err != nil {
+			return fmt.Errorf("while creating k8s client: %w", err)
+		}
+		secrets := &v1.SecretList{}
+		listOpts := []client.ListOption{
+			client.InNamespace("kyma-system"),
+			client.MatchingFields{"metadata.name": "sap-btp-manager"},
```

Comment:
Why do we select by `metadata.name` not just by name?

Reply:
I had some issues when testing it with name and no secrets were returned.

Reply:
Fixed, so we now get the secret by name.